app/src/organisms/PipetteWizardFlows/Results.tsx

```diff
@@ -59,11 +67,21 @@ export const Results = (props: ResultsProps): JSX.Element => {
     }
     case FLOWS.ATTACH: {
       // attachment flow success
-      if (attachedPipettes[mount] != null) {
+      if (
+        (attachedPipettes[mount] != null && requiredPipette == null) ||
+        Boolean(isCorrectPipette)
+      ) {
         header = t('pipette_attached', { pipetteName: pipetteName })
         buttonText = t('cal_pipette')
-        // attachment flow fail
+        // attached wrong pipette
+      } else if (
+        attachedPipettes[mount] != null &&
+        Boolean(!isCorrectPipette)
+      ) {
+        header = i18n.format(t('wrong_pip'), 'capitalize')
+        buttonText = i18n.format(t('detach_and_retry'), 'capitalize')
```

Comment:
We need to change `iconColor` and  `isSuccess` fields so that this isn't shown as a green check mark
```suggestion
        buttonText = i18n.format(t('detach_and_retry'), 'capitalize')
        iconColor = COLORS.errorEnabled
        isSuccess = false

Reply:
oops thanks!